docs/api/web-contents.md

```diff
@@ -1797,6 +1798,10 @@ Returns `boolean` - Whether the devtools is opened.
 
 Returns `boolean` - Whether the devtools view is focused .
 
+#### `contents.getDevToolsTitle()`
+
+Returns `string` - the title of devtools .
+
```

Comment:
```suggestion
#### `contents.getDevToolsTitle()`

Returns `string` - the current title of the devtools window. This will only be visible if devtools is been open in `undocked` or `detach` mode.

```

Reply:
I also think we'd want to document a separate setter function - `contents.setDevToolsTitle()` or similar. Being exclusively able to set it in `webContents.openDevtools()` feels somewhat clunky.